MangoAPI.Client/src/app/shared/blob-storage.service.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable } from '@angular/core';
+import { BlobServiceClient, ContainerClient } from '@azure/storage-blob';
+import { Observable } from 'rxjs';
+import { environment } from 'src/environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class BlobStorageService {
+
+  accountName = "devstoreaccount1";
+  containerName = "mangocontainer";
+  sas = environment.sasToken;
+
+  constructor() {
+  }
+
+
+  public async listImages(): Promise<string[]> {
```

Comment:
Blob Storage service is already implemented on backend, so it is better to use it instead creating new one on frontend

See https://github.com/MangoInstantMessenger/MangoMessengerAPI/blob/develop/MangoAPI.Application/Services/BlobService.cs